core/modules/date/date.admin.inc

```diff
@@ -557,7 +557,7 @@ function _date_field_settings_form($field, $instance, $has_data) {
   $form['cache_enabled'] = array(
     '#type' => 'checkbox',
     '#title' => t('Cache dates'),
-    '#description' => t('Date objects can be created and cached as date fields are loaded rather than when they are displayed to improve performance.'),
+    '#description' => t('Create and cache date objects when date fields are being loaded, rather than when they are being displayed. This improves performance.'),
```

Comment:
Maybe...
```suggestion
    '#description' => t('Create and cache date objects when date fields are being loaded, rather than when they are being displayed. This can help to improve performance.'),
```